index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
I'm a little confused here. Shouldn't the `audience` be the origin URL of the sender?

I wrote this code following https://blog.mozilla.org/services/2016/04/04/using-vapid-with-webpush/, but now reading the specification (https://tools.ietf.org/html/draft-thomson-webpush-vapid-02) it seems like `audience` should be the origin of the push resource URL...

@jrconlin